server/util/grpc_client/grpc_client.go

```diff
@@ -66,5 +68,18 @@ func CommonGRPCClientOptions() []grpc.DialOption {
 		filters.GetUnaryClientInterceptor(),
 		filters.GetStreamClientInterceptor(),
 		grpc.WithDefaultCallOptions(grpc.MaxCallRecvMsgSize(math.MaxInt32)),
+		grpc.WithKeepaliveParams(keepalive.ClientParameters{
+			// After a duration of this time if the client doesn't see any activity it
+			// pings the server to see if the transport is still alive.
+			Time: 10 * time.Second,
```

Comment:
Might want to set this to something a little higher than 10s, like 20s - because our grpc server enforcement policy is 10s.

https://github.com/buildbuddy-io/buildbuddy/blob/393e82020e9406dada2edb4708225b010c3f42ac/server/util/grpc_server/grpc_server.go#L88

Reply:
Done